mabwiser/linear.py

```diff
@@ -106,10 +106,12 @@ class _Linear(BaseMAB):
     factory = {"ts": _LinTS, "ucb": _LinUCB, "ridge": _RidgeRegression}
 
     def __init__(self, rng: _BaseRNG, arms: List[Arm], n_jobs: int, backend: Optional[str],
-                 l2_lambda: Num, alpha: Num, regression: str, arm_to_scaler: Optional[Dict[Arm, Callable]] = None):
+                 l2_lambda: Num, alpha: Num, epsilon: Num, regression: str,
```

Comment:
Since all the linear policies require `l2_lambda` this ordering makes more sense to me. Is it worth changing API to LinUCB(l2_lambda, alpha, ...), LinTS(l2_lambda, alpha, ..) and LinGreedy(l2_lambda, epsilon)?

Reply:
Actually ignore what I said, I'm not sure if putting l2_lambda first makes more sense.

Reply:
Based on the discussion, so the sequence in the __init__ parameters would be alpha, epsilon, l2_lambda, regression, etc?

Reply:
Yes, changing ordering to _Linear(... alpha, epsilon, l2_lambda, regression, ...) would be "more consistent" with public API I think.
@skadio - Do you have another suggestion?

Reply:
What's the most common param for all? I would put that one first, and so-on. 

BUT if this leads to a change in the Public API (that is it would break others/users code in subtle ways in the next pip install (and they won't notice it since these params interoperate due to common 0-1 range --then I am not inclined to make any changes.

Reply:
Makes sense. l2_lambda is the common param for all the linear policies, BUT putting that first will break public API.

@wddcheng - Let's just change internal _Linear to at least have partial ordering consistent with public API, as you suggested above.